comps/embeddings/langchain-mosec/docker/README.md

```diff
@@ -0,0 +1,43 @@
+# Embedding Server
+
+## 1. Introduction
+
+This is a replacement for OpenAI embedding.
```

Comment:
shall you specify this docker image could only run on Xeon CPU?

Reply:
OK, I will change the introduction to highlight this image is only used for Xeon CPU.